google-cloud-video_intelligence/google-cloud-video_intelligence.gemspec

```diff
@@ -20,7 +20,9 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = ">= 2.0.0"
 
-  gem.add_dependency "google-gax", "~> 0.10.1"
+  gem.add_dependency "google-gax", "~> 1.0"
+  gem.add_dependency "googleapis-common-protos", "~> 1.3.1"
+  gem.add_dependency "googleauth", "~> 0.6.1"
```

Comment:
I don't like setting these dependency if it can be avoided, which I believe it can. What happens if google-gax 1.1.0 is released and changes the dependencies to `googleapis-common-protos ~> 1.4.0` and `googleauth ~> 0.7.0`? That would cause dependency conflicts that could otherwise be avoided.

Reply:
I agree with this. I am checking now to see why these were added to the autogen templates.

Reply:
Ah, it looks like maybe these were always there and the reason we're seeing them again in the diff is that you manually removed them in https://github.com/geigerj/google-cloud-ruby/commit/8682c7d8f0849e347ece239f377a51a80765b7f5#diff-0ee68bc1b35563909430d4b42adbcce1.

Reply:
Filed: https://github.com/googleapis/toolkit/issues/1752. For future reference, please tag me when editing autogen files so I know to get the appropriate modifications propagated back into the generator.